common/Cargo.toml

```diff
@@ -39,6 +39,7 @@ tracing-subscriber = { workspace = true, optional = true }
 typeshare = { workspace = true }
 url = { workspace = true, features = ["serde"] }
 uuid = { workspace = true, features = ["v4", "serde"], optional = true }
+utoipa = { version = "5", features = ["chrono"], optional = true }
```

Comment:
For clarity/visibility, I would also add `utoipa = ["dep:utoipa"]` under features, like tonic has.

Reply:
:+1:  Addressed in d81ddcb9b5a8